src/App/Home.js

```diff
@@ -0,0 +1,57 @@
+/* eslint-disable consistent-return */
+import React, { Component } from 'react';
+import ReactResizeDetector from 'react-resize-detector';
+import WideAboutUs from './Homepage/Widescreen/WideAbout';
+import WideCurrentProjects from './Homepage/Widescreen/WideCurrentProjects';
+import WideTicker from './Homepage/Widescreen/WideFacebookTicker';
+import NarrowAboutUs from './Homepage/Narrowscreen/NarrowAbout';
+import NarrowCurrentProjects from './Homepage/Narrowscreen/NarrowCurrentProjects';
+import NarrowTicker from './Homepage/Narrowscreen/NarrowFacebookTicker';
+
+// eslint-disable-next-line react/prefer-stateless-function
+export default class Home extends Component {
+  constructor(props) {
+    super(props);
+    this.parentRef = React.createRef();
+    this.onResize = this.onResize.bind(this);
+    this.state = { width: 100 };
+  }
+
+  onResize(width) {
+    this.setState({ width });
```

Comment:
similarly, you could use the height to set the placement of the footer to be at the bottom of the page if you want, or you can implement a different solution if you wish

Reply:
Would fix it for Home.js, but if I'm not mistaken, it'd still probably be an issue elsewhere.

I'm going to try using flexbox to fix it site wide ^^